src/connection/database/index.js

```diff
@@ -136,7 +136,7 @@ function processDatabaseOptions(opts) {
         prefill = undefined;
       }
 
-      const types = ['select', 'text', 'checkbox'];
+      const types = ['select', 'text', 'checkbox', 'hidden'];
```

Comment:
Would it be easy to add `email`, `url`, and `password` here? Would be nice to have that built-in browser validation.

Reply:
that's another feature request 😬